From zaitcev@redhat.com Fri Nov 11 01:19:22 2005
Date: Fri, 11 Nov 2005 01:14:36 -0800
From: Pete Zaitcev <zaitcev@redhat.com>
To: greg@kroah.com, Adrian Bunk <bunk@stusta.de>
Cc: stern@rowland.harvard.edu, akpm@osdl.org, mdharm-usb@one-eyed-alien.net, zaitcev@redhat.com, <reuben-lkml@reub.net>
Subject: USB: Makefile change for libusual
Message-Id: <20051111011436.5804aac8.zaitcev@redhat.com>

Never one to ignore a good idea, I propose to take a segment of
Adrian's patch which addresses this particular problem:

> --- linux-2.6.14-mm1-full/drivers/usb/Makefile.old	2005-11-08 01:31:00.000000000 +0100
> +++ linux-2.6.14-mm1-full/drivers/usb/Makefile	2005-11-08 01:31:26.000000000 +0100
> @@ -22,6 +22,7 @@
>  obj-$(CONFIG_USB_PRINTER)	+= class/
>  
>  obj-$(CONFIG_USB_STORAGE)	+= storage/
> +obj-$(CONFIG_USB_LIBUSUAL)	+= storage/
>  
>  obj-$(CONFIG_USB_AIPTEK)	+= input/
>  obj-$(CONFIG_USB_ATI_REMOTE)	+= input/

With a small change, this seems to work fine with the supplied .config.
Also I have tested my usual build modes and added Reuben's configuration
to my testing list.

Signed-off-by: Pete Zaitcev <zaitcev@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/Makefile |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/Makefile
+++ gregkh-2.6/drivers/usb/Makefile
@@ -20,6 +20,7 @@ obj-$(CONFIG_USB_ACM)		+= class/
 obj-$(CONFIG_USB_PRINTER)	+= class/
 
 obj-$(CONFIG_USB_STORAGE)	+= storage/
+obj-$(CONFIG_USB)		+= storage/
 
 obj-$(CONFIG_USB_AIPTEK)	+= input/
 obj-$(CONFIG_USB_ATI_REMOTE)	+= input/
